tornado-api/src/main/java/uk/ac/manchester/tornado/api/TaskSchedule.java

```diff
@@ -70,8 +70,8 @@
 /**
  * Tornado Task Schedule API.
  * <p>
- * </> Task-based parallel API to express code to be compiled and executed from
- * Java to OpenCL a t runtime. The Tornado runtime executes the generated OpenCL
+ * Task-based parallel API to express code to be compiled and executed from Java
+ * to OpenCL a t runtime. The Tornado runtime executes the generated OpenCL
```

Comment:
this comment needs to be reworked to add PTX and SPIRV?

Reply:
Yes, I will update this